AdofaiTweaks/Tweaks/KeyViewer/KeyViewer.cs

```diff
@@ -346,5 +346,14 @@ internal class KeyViewer : MonoBehaviour
         private static string TweenIdForKeyCode(KeyCode code) {
             return $"adofai_tweaks.key_viewer.{code}";
         }
+
+        public void ClearCounts()
```

Comment:
Can you add a doc comment here? I have an analyzer that shows a warning for any undocumented public methods, which I like to have on.

Reply:
Function documented,. Apparently my IDE warned me about the undocumented function, but I didn't read the warning...